xblockgrademe/private/view.js

```diff
@@ -4,8 +4,32 @@ function XblockGrademeView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
 
-    // TODO: Put your logic here
+    var failure_message = "We are sorry; users who have not created accounts and registered " +
+                          "for the course may not receive a Statement of Accomplishment.";
+    
+    var success_message = "Your grading request has been received. If you have passed, your Statement of " +
+                          "Accomplishment should be available on your <a href='/dashboard'>account dashboard</a> " +
+                          "page within the next 20 minutes.";
 
-    // To find elements inside your XBlock, try:
-    // var myElement = $element.find('.myElement');
+    var username = $('li.primary a.user-link').text();
+
+    var scraped_username = $('li.primary a.user-link').text();
+    if (scraped_username) {
+        document.getElementById('User_Verify_Button').addEventListener("click",
+        function(event) {
+            (function(event) {
+                $.ajax({
+                    type:     'POST',
+                    url:      '/request_certificate',
+                    data:     {'course_id': $$course_id},
+                    success:  function(data) {
+                        $('.xblockgrademe_block .verify-button-message-text').html(success_message);
+                    }
+                });
+            }).call(document.getElementById('User_Verify_Button'), event);
```

Comment:
Nope, this is how Joe implemented this, and as this is supposed to be a pretty straightforward port of the "Grade Me" html block, I think I'd prefer to leave it as is for v0?